src/wallet/walletutil.cpp

```diff
@@ -43,4 +43,10 @@ WalletFeature GetClosestWalletFeature(int version)
     }
     return static_cast<WalletFeature>(0);
 }
+
+bool IsSolvable(const CScript& script, const SigningProvider& provider) {
```

Comment:
I don't see why introducing the new `IsSolvable`, with swapped arguments, is clearer than ripping out the old one at the same time. Last two commits can be same commit imo

Reply:
There were more call sites before the rebase. Agree we can actually drop the new `IsSolvable` helper now.

Reply:
Done

Reply:
Looks like you nuked the old definition, a bit more than I asked for  :P

Reply:
Yes, it was only used once in tests, i didn't see the need for keeping the helper then :)